docs/pages/framework/models/index.md

```diff
@@ -147,16 +154,28 @@ mod spawnHuman {
 
             // spawn a human
```

Comment:
Actually on this example, an auditor mentioned that we have an issue. Since the counter for human and goblin starts at the same value, they will share the same storage location.

We should hash the id of the goblin / human with a constant like 'HUMAN', 'GOBLIN'.
Because `Position` with `id = 1` will not distinguish between human and goblin.

Would you mind adjusting it since you're already adjusted the example?

Reply:
Ah yes, nice catch ! 👍